package.json

```diff
@@ -31,10 +31,13 @@
   },
   "homepage": "https://github.com/codecov/uploader#readme",
   "dependencies": {
+    "@types/node-fetch": "2.5.12",
```

Comment:
This should be in `devDependencies`.

Reply:
> Do we actually look for codecov.yml in the .github folder?

Yes, see https://docs.codecov.com/docs/codecov-yaml#can-i-name-the-file-codecovyml